build/ci_utils/src/cache/goodie/graalvm.rs

```diff
@@ -171,9 +159,30 @@ Java(TM) SE Runtime Environment Oracle GraalVM 17.0.7+8.1 (build 17.0.7+8-LTS-jv
 Java HotSpot(TM) 64-Bit Server VM Oracle GraalVM 17.0.7+8.1 (build 17.0.7+8-LTS-jvmci-23.0-b12, mixed mode, sharing)";
 
         let found_graal = graal_version_from_version_string(version_string).unwrap();
-        assert_eq!(found_graal, Version::new(23, 0, 0));
+        let expected_graal_version = Version {
+            major: 17,
+            minor: 0,
+            patch: 7,
+            pre: Prerelease::EMPTY,
+            build: BuildMetadata::new("8.1").unwrap()
+        };
+        assert_eq!(found_graal, expected_graal_version);
 
         let found_java = Java.parse_version(version_string).unwrap();
         assert_eq!(found_java, Version::new(17, 0, 7));
     }
+
+    #[test]
+    fn recognize_oneline_version() {
+        let version_line = "Java(TM) SE Runtime Environment Oracle GraalVM 17.0.7+8.1 (build 17.0.7+8-LTS-jvmci-23.0-b12)";
```

Comment:
Got tricked, Pavel? If it says `Oracle GraalVM`, then it is not the open source version, I think.

Reply:
Yes, I got confused a bit. I think that I understand it now - The following output is from the "new EE version" (which is now free for testing purposes, this might be the modification, as I believe the older versions could only be downloaded via Oracle Technology Network?):
```
java version "17.0.7" 2023-04-18 LTS
Java(TM) SE Runtime Environment Oracle GraalVM 17.0.7+8.1 (build 17.0.7+8-LTS-jvmci-23.0-b12)
Java HotSpot(TM) 64-Bit Server VM Oracle GraalVM 17.0.7+8.1 (build 17.0.7+8-LTS-jvmci-23.0-b12, mixed mode, sharing)
```

And this is the output from the "new CE version":
```
openjdk version "17.0.7" 2023-04-18
OpenJDK Runtime Environment GraalVM CE 17.0.7+7.1 (build 17.0.7+7-jvmci-23.0-b12)
OpenJDK 64-Bit Server VM GraalVM CE 17.0.7+7.1 (build 17.0.7+7-jvmci-23.0-b12, mixed mode, sharing)
```

So nowadays, `Oracle GraalVM` == enterprise edition and `GraalVM CE` == community edition.

Reply:
I got "tricked" because the [official download page](https://www.graalvm.org/downloads/#) downloads the enterprise edition, and if you want to download the community edition, you have to click on "GraalVM Community Edition" under "Installation Options", which is not as intuitive as I would like.